src/utils/setNumber.ts

```diff
@@ -0,0 +1,29 @@
+import {Validation} from './Validation';
+
+export default function setNumber(
+	limit: number,
+	setter: (value: string) => void,
+	value: string,
+	isMonth: boolean,
+	isNumber: boolean,
+) {
```

Comment:
많은 prop가 다음과 같이 구현되어있어서 사용하는 곳에서는 몇 번째 props가 어떤 값인지 한 번에 확인하기 힘든것 같아요.
props들을 객체로 묶어서 명시적으로 어떤 값에 무슨 값이 들어가는 지 알 수 있도록 해보는 건 어떨까요?
```suggestion
interface exampleProps {
	limit: number,
	setter: (value: string) => void,
	value: string,
	isMonth: boolean,
	isNumber: boolean,
}

export default function example({
	limit,
	setter,
	value,
	isMonth,
	isNumber,
}: exampleProps) {
```

Reply:
수정했습니다. 어떤 파라미터에 어떤 인자를 넘기는지 알기 쉬워졌습니다 👍

(setNumber는 컴포넌트가 아니라서 prop를 받지 않습니다!)